recipes/gemmi/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+mkdir -p build
+cd build
+
+cmake .. \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release
+
+make -j$CPU_COOUNT
+
+cp gemmi "$PREFIX/bin/"
```

Comment:
Why is this needed if you use CMAKE_INSTALL_PREFIX?

Reply:
There's no `install` command in the cmake config, so it has to be done manually. So `CMAKE_INSTALL_PREFIX` probably isn't doing anything, I just added it out of habit.